src/chttpd/src/chttpd_changes.erl

```diff
@@ -466,7 +466,12 @@ send_changes_doc_ids(Db, StartSeq, Dir, Fun, Acc0, {doc_ids, _Style, DocIds}) ->
 
 
 send_changes_design_docs(Db, StartSeq, Dir, Fun, Acc0, {design_docs, _Style}) ->
-    FoldFun = fun(FDI, Acc) -> {ok, [FDI | Acc]} end,
+    FoldFun = fun(FDI, Acc) ->
+        case FDI of
+            {row, Row} -> {ok, [fabric2_db:get_full_doc_info(Db, proplists:get_value(id, Row)) | Acc]};
```

Comment:
Let's make sure the line is shorter than 80 columns if possible.

Reply:
Done in 343015774